main_test.go

```diff
@@ -18,3 +23,39 @@ func TestGORM(t *testing.T) {
 		t.Errorf("Failed, got error: %v", err)
 	}
 }
+
+var sentences = []string{
+	"Can",
+	"You",
+	"Please",
+	"Tell",
+	"What",
+	"Am",
+	"Doing",
+	"Wrong",
+}
+
+func TestInsertInSqliteAsync(t *testing.T) {
+	conn := SetupConnection()
+	db, _ := conn.DB()
+	defer db.Close()
+	errChannel := make(chan error)
+	for _, s := range sentences {
+		// Implemented waitgroups because i want to log all errors
+		// the channel will close after all the goroutines have run.
+		wg.Add(1)
+		go insert(conn, errChannel, Sentence{ActualSentence: s})
+	}
+	go func() {
+		wg.Wait()
```

Comment:
hello, should not use go func in here, maybe cause db closed, but insert action still run.
```
wg.Wait()
close(errChannel)
```